File: sdks/php/src/Providers/XAI.php
<?php

declare(strict_types=1);

namespace CyberArmor\Providers;

use CyberArmor\Client;

final class XAI extends OpenAICompatible
{
    public function __construct(Client $client, object $openai, string $tenantId, string $model = 'grok-3')
    {
        parent::__construct($client, $openai, $tenantId, 'xai', $model);
    }
}

File: sdks/ruby/lib/cyberarmor/providers/xai.rb
# frozen_string_literal: true

module CyberArmor
  module Providers
    class XAI < OpenAICompatible
      def initialize(client:, openai:, tenant_id:, model: 'grok-3')
        super(client: client, openai: openai, tenant_id: tenant_id, provider: 'xai', model: model)
      end
    end
  end
end

